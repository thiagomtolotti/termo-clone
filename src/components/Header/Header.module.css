.superior {
  display: flex;
  flex-direction: column;
}

.header {
  display: flex;
  justify-content: center;
}

.header h1 {
  letter-spacing: 2px;
}
