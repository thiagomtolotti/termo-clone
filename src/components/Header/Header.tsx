export const Header = () => {
  return <h1>Termo</h1>;
};
