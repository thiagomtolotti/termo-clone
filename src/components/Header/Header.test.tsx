import { render } from "@testing-library/react";
import { Header } from "./Header";

describe("Component Header", () => {
  it("Should render correctly", () => {
    const renderResult = render(<Header />);

    expect(renderResult.container).toMatchSnapshot();
  });
});
