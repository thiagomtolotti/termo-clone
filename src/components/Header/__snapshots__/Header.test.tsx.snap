// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header Should render correctly 1`] = `
<div>
  <header
    class="header"
  >
    <h1>
      Termo
    </h1>
  </header>
</div>
`;
